### PR TITLE
FEM: mesh: XML fenics mesh export fixed

### PR DESCRIPTION
**Pull request by joha2**
XML fenics mesh export was broken, due to an API change for the FreeCAD.Console.

This small PR fixes the issue. @qingfengxia provided the fix which was discussed in https://forum.freecadweb.org/viewtopic.php?f=18&t=35778&sid=d574cf84b7474a3941f601066d5a9b76&start=20

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [X] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [X] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Fem/feminout/writeFenicsXML.py [hunk 1]
```diff
@@ -68,17 +68,17 @@ def write_fenics_mesh_xml(fem_mesh_obj, outputfile):
         "hexahedron": 8
     }
 
-    Console.Message("Converting " + fem_mesh_obj.Label + " to fenics XML File\n")
-    Console.Message("Dimension of mesh: %d\n" % (get_FemMeshObjectDimension(fem_mesh_obj),))
+    Console.PrintMessage(u"Converting " + fem_mesh_obj.Label + u" to fenics XML File\n")
```
Comment (vocx-fc) [inline src/Mod/Fem/feminout/writeFenicsXML.py]: Why do you need to prepend with `u"string"`? I don't think this is necessary. Maybe it was used before, with Python 2, but it's not required anymore.

Comment (joha2) [reply src/Mod/Fem/feminout/writeFenicsXML.py]: @vocx-fc thanks for pointing this out. We were just not sure about that. What do we have to do to maintain backwards compatibility?

Comment (vocx-fc) [reply src/Mod/Fem/feminout/writeFenicsXML.py]: Mmm. Not sure, but maybe nothing? I mean, you should only add the `u""` if you really need it. Do you notice any difference if you don't add the `u""`? If not, then don't add it.

Comment (yorikvanhavre) [reply src/Mod/Fem/feminout/writeFenicsXML.py]: In python3 (which can now be considered FreeCAD default, as we are slowly abandonning py2 support), all strings are unicode. So u"Something" is fully equal to "Something". I would also remove the u I guess

**Comment by berndhahnebach** [comment]
merged and deleted u as it is not used in FEM console messages at all.

**Comment by joha2** [comment]
Thanks @berndhahnebach for merging. :smile:

**Pull request closed without merging**